src/html.ejs

```diff
@@ -36,7 +36,8 @@
     <%- htmlWebpackPlugin.tags.headTags.map((originalTag) => { const tag = { ...originalTag, attributes: { ...originalTag.attributes } }; tag.attributes['src'] = "<" + "%= cdnUrl %" + ">" + originalTag.attributes['src']; return tag; }) %>
   <%% } %>
 
-  <%%- content.linkPreloadTags.join("") %>
+  <%%- content.imageLinkPreloadTags.join("") %>
```

Comment:
Won't this duplicate the `<link>` elements rendered within the body ([e.g.](https://github.com/artsy/force/blob/7ec3ee23f5b3c11582d92bf9ea5494525db18ee9/src/Apps/Artist/Components/ArtistHeader/ArtistHeaderImage.tsx#L36))?

Reply:
Ah yep, i was thinking we can remove this stuff now since its now all automated.

Reply:
fixed